packages/harpguru-core/src/components/harp-face-fragment/hooks/use-styles.ts

```diff
@@ -15,8 +14,8 @@ export const useStyles = (
   harpfaceIndex: 'harpface1' | 'harpface2'
 ): HarpFaceFragmentStyles => {
   const {
-    dynamicSizes: { columnWidth, rowHeight },
-  } = useSizes()
+    [0]: { columnWidth, rowHeight },
+  } = useGlobal('dynamicSizes')
```

Comment:
Similar point to https://github.com/js-jslog/harpguru/pull/137#discussion_r765155671

Reply:
Done